biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
I agree it's not trivial. However, we know for a fact that we have lost many MINOR changes because of this issue.

Case in point:
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.liferay%22%20AND%20a%3A%22com.liferay.wiki.service%22

Between version 1.2.0 to current version (1.2.18) an exception was removed from a method call, which should have caused the version to go from 1.2.x to 1.3.0 because of the API change. However it clearly has not. The change was effectively lost.